src/Symfony/Component/DependencyInjection/Attribute/Inner.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Attribute;
+
+#[\Attribute(\Attribute::TARGET_PARAMETER)]
+class Inner
```

Comment:
`DecoratedService`?

Reply:
As you want. I choose `Inner` to keep the same naming as the `@.inner` service used in yaml configuration.

Reply:
`InnerService`?

Reply:
:ok_hand: changed !